pyomo/contrib/doe/fim_doe_tutorial_parmest.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
This notebook should be divided into sections: (1) defining the model, (2) regressing the model parameters with Parmest, (3) performing MBDoE analysis with Pyomo.DOE. This notebook will become a tutorial for new Pyomo.DOE users.